src/graphql/mutations.js

```diff
@@ -110,14 +110,20 @@ export const SET_REGISTRANT = gql`
 `
 
 export const COMMIT = gql`
-  mutation commit($label: String, $owner: String, $secret: String) {
-    commit(label: $label, owner: $owner, secret: $secret) @client
```

Comment:
Same as comment below

Reply:
Same answer as above. The secret is not taken from the UI, so it doesn't need to be part of the mutation